src/content/docs/apm/agents/python-agent/supported-features/python-message-queues.mdx

```diff
@@ -71,21 +44,55 @@ Supported entry points for queue operations appear as `Put` (publish a message)
       </td>
 
       <td>
-        `basic_publish`
+        2.88.0.72
+      </td>
+    </tr>
+        <tr>
+      <td>
+        Kafka Python
       </td>
 
       <td>
-        `basic_get`
+        8.2.0.181
+      </td>
+    </tr>
+        <tr>
+      <td>
+        Confluent Kafka
+      </td>
 
-        `basic_consume (callback)`
+      <td>
+        8.2.0.181
       </td>
     </tr>
   </tbody>
 </table>
 
+For more information, see [the release notes](/docs/release-notes/agent-release-notes/python-release-notes).
+
+<Callout variant="caution">
+  RabbitMQ Message tasks are not started for consumers using [TornadoConnection](https://pika.readthedocs.io/en/latest/modules/adapters/tornado.html#pika.adapters.tornado_connection.TornadoConnection).
+</Callout>
+
+## Performance improvements with background tasks [#background]
+
+One way to increase responsiveness of web applications is to delegate work to background processes. Message queues are commonly used for this inter-process communication.
+
+In the context of message queuing systems, applications typically interact with message brokers to send and receive messages. In the case of RabbitMQ Pika, this client library allows Python applications to interface with message brokers that implement the Advanced Message Queueing Protocol (AMQP) 0.9 or higher.
+
+The Python agent shows messages sent and received using the client libraries. With this visibility, you can see details including:
+
+* Number of messages produced by your app
+* Time your app spends publishing/producing messages
+* Time your app spends processing/consuming messages
+
+[APM](#view-queue) conveniently groups and reports operations that interact with queues. By analyzing this information, you can more easily identify bottlenecks and areas for performance improvement in your message passing architecture.
+
 ## View in New Relic UI [#view-queue]
 
-Queue operations appear on APM's [**Transactions** page](/docs/apm/applications-menu/monitoring/transactions-dashboard) for the selected app. The `Put` and `Take` metrics appear in the **Breakdown table** and are categorized as `MessageBroker` metrics. Here is an example:
+Queue operations appear on APM's [**Transactions** page](/docs/apm/applications-menu/monitoring/transactions-dashboard) for the selected app. 
+
+The `Put` and `Take` metrics appear in the **Breakdown table** and are categorized as `MessageBroker` metrics for RabbitMQ. Here is an example:
```

Comment:
I think we should clarify somewhere what `Put` and `Take` translate to in terms of kafka (`Produce` and `Consume`).

Reply:
Since I am pointing to an example for RabbitMQ, I'll add framework agnostic clarification to the terms, but I will add a table before this section that will clarify what terms correspond with what brokers.